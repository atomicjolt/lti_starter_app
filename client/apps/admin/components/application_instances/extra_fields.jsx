
export default function getExtraFields(appKey) {
  if (appKey === 'search' || appKey === 'navigator') {
    return [{

    }];
  }
  return [];
}
