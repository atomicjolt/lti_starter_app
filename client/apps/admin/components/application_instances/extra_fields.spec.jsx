import React from 'react';
import TestRenderer from 'react-test-renderer';

import { extra_fields } from './extra_fields';

describe('extra_fields', () => {
  it('renders the extra_fields component', () => {
    const result = TestRenderer.create(
        <extra_fields />
    );
    expect(result).toMatchSnapshot();
  });
});
