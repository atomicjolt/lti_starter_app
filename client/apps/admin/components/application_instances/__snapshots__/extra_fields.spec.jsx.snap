// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`extra_fields renders the extra_fields component 1`] = `<extra_fields />`;
